### ISSUE TITLE
Showing local time with seconds in notification center calendar

### ISSUE THREAD

**Issue by chk1** [ramensoftware/windows-11-notification-center-styling-guide#3]
One thing I've been missing in Windows 11 is the display of time with seconds in the notification center / calendar popup. I know there's an option to show the seconds in the taskbar clock (with and without Windhawk mods), but that's not what I want because I find that too distracting. 

I did a quick search around with UWPSpy and landed on this `TextBlock` element:

![image](https://github.com/ramensoftware/windows-11-notification-center-styling-guide/assets/828496/ef758dcb-23c7-4432-ac2d-c7c4b0a9563b)

Is there currently an easy way to just add the current time there? I had a look at the "Taskbar Clock Customization" mod and it source code, and it seems to be more involved than just setting a time format value somewhere in some UWP field attribute. 

There's a question on Superuser asking basically the same: https://superuser.com/questions/1801741/display-windows-11-clock-like-the-one-in-windows-10-when-tray-is-clicked

**Comment by bbmaster123** [comment]
no, that's unfortunately not possible with the styler mod, as far I am aware. It might be possible with a mod designed to do that specifically. This is more to change the default parameters that are already there.

If I may suggest, as its the only thing I can think of, you could try using explorerpatcher for its option to use win+c to open the windows 10 clock, which overrides opening copilot.

actually, now that I think about it, you might also be able to use the taskbar styler or the clock customization mod to only show seconds on hover, if that's something you'd be interested in.

Sorry I can't be of more help

**Comment by chk1** [comment]
While updating my mods today, I noticed that the "Windows 11 Notification Center Styler" mod had screenshots with full clock incl. seconds. And there was a new reply to the Superuser question I linked above:

> Microsoft brought back the ability to show the time with seconds in the Notification Center for Windows 11 [...]. Enable this feature by opening Settings > Time & language > Date & time, and turn on “Show time in Notification Center.”

Result:
<img width="352" height="143" alt="Image" src="https://github.com/user-attachments/assets/dc074311-a8b1-41a0-a67f-6545d7876fd0" />

So this works now :)
